src/gen_rpc_app.erl

```diff
@@ -35,7 +35,8 @@ stop(_State) ->
 %%% ===================================================
 -spec start() -> 'ok' | {'error', term()}.
 start() ->
-    application:start(?APP).
+    application:ensure_all_started(?APP),
+    start(normal, []).
```

Comment:
This should never be called when running the app :) I should remove these lines altogether. Why did you update them? I mean where did you use that function?